app/tests/unit/containers/Applications/ApplicationReviewValidationContainer.test.tsx

```diff
@@ -0,0 +1,148 @@
+import {ApplicationReviewValidation} from 'containers/Applications/ApplicationReviewValidationContainer';
+import {mount} from 'enzyme';
+import React from 'react';
+import {ApplicationReviewValidationContainer_applicationRevision} from '__generated__/ApplicationReviewValidationContainer_applicationRevision.graphql';
+
+describe('The ApplicationReviewValidationContainer', () => {
+  it('Matches the snapshot with failed validations', () => {
+    const appRev: ApplicationReviewValidationContainer_applicationRevision = {
+      ' $refType': 'ApplicationReviewValidationContainer_applicationRevision',
+      overrideJustification: undefined,
+      validation: {
+        edges: [
+          {
+            node: {
+              isOk: false,
+              validationDescription: 'this is one failed test validation'
+            }
+          },
+          {
+            node: {
+              isOk: false,
+              validationDescription: 'this is another failed test validation'
+            }
+          },
+          {
+            node: {
+              isOk: true,
+              validationDescription: 'this is a passing validation'
+            }
+          }
+        ]
+      }
```

Comment:
To avoid copying and pasting test queries into multiple tests, then having to carefully update every one of them if and when the fragment query changes, in exactly the right way while making sure it still tests the intended case, in all of the new [application review components I've been using a function for this](https://github.com/bcgov/cas-ciip-portal/blob/d36695f4b2517d41a62a695b6c55ccc937f2f353/app/tests/unit/containers/Admin/ApplicationReview/ApplicationReviewStepSelector.test.tsx#L7):
```
const data = getTestQuery(options);
```

For components that have many props, I've also outsourced instantiation of the test element to a function.